packages/label/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "label",
+  "version": "1.0.0",
+  "description": "Little Spoon label",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "build:flowtypes": "flowgen lib && rm -rf lib && mv exports lib && yarn build",
+    "clean": "rm -rf coverage exports lib",
+    "lint": "eslint .",
+    "lint:fix": "yarn lint --fix",
+    "lint:tsc": "tsc --noEmit",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {},
```

Comment:
You'll need to add `@littlespoon/theme` to dependencies to fix the build failures

Reply:
Added it in the dependencies.